front/src/applications/operationalStudies/views/ManageTrainSchedule.tsx

```diff
@@ -179,6 +179,7 @@ export default function ManageTrainSchedule() {
     if (shouldRunPathfinding && mustUpdatePathfinding === false) {
       setMustUpdatePathfinding(true);
     }
+    setMustUpdatePathfinding(false);
```

Comment:
It seems that with this extra line, the pathfinding is not relaunched if we change the origin or destination in the pathfinding tab

Reply:
True, thank you for your test :pray: I have tried something else, can you review and test again please ? Thanks ! :)